Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/RideBooking/API/Driver.yaml

```diff
@@ -239,6 +239,10 @@ types:
     - totalAcRestrictionUnblockCount: Int
     - lastACStatusCheckedAt: Maybe UTCTime
     - currentACStatus: Bool
+    - assignedCount : Maybe Int
+    - cancelledCount : Maybe Int
+    - cancellationRate : Maybe Int
+    - windowSize : Maybe Int
```

Comment:
Instead of having all fields we can directly have "CancellationRateData" from module "SharedLogic.CancellationRate" in DriverInfoRes.